regparser/commands/tests.py

```diff
@@ -0,0 +1,51 @@
+import click
+import importlib
+import nose
+import os
+import requests_cache
+from stevedore import extension
+from stevedore.exception import NoMatches
+
+
+def get_stevedore_module_names(namespace):
+    names = []
+
+    def extension_adder(ext):
+        if len(ext.entry_point.attrs) == 0:  # account for case without :
+            test_name = ext.entry_point.module_name
+        else:
+            test_name = ext.entry_point_target
+        names.append(test_name)
+
+    try:
+        stevedore_mgr = extension.ExtensionManager(
+            namespace=namespace, invoke_on_load=False)
+        stevedore_mgr.map(extension_adder)
+    except NoMatches:
+        pass
+
+    return names
+
+
+@click.command()
+def tests():
```

Comment:
I worry that `tests` is too generic. `test_with_plugins`? I don't know.

Docstrings are particularly important for `click` commands -- they're used to derive the help text for the command line tool.